runtime/doc/news.txt

```diff
@@ -174,7 +174,9 @@ TUI
 
 UI
 
-• TODO
+• The |ui-popupmenu| now supports cascading highlight styles. |hl-PmenuSel|
+  and |hl-PmenuMatch| both inherit from |hl-Pmenu|, and |hl-PmenuMatchSel|
+  inherits highlights from both |hl-PmenuSel| and |hl-PmenuMatch|.
```

Comment:
let's mention this at `:help hl-PmenuSel`, etc.

also in the `Highlight groups` section of `:help nvim-features`.